### PR TITLE
datastore/tasks: add system tests

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

**Comment by broady** [comment]
@okdave 
@adg 

PTAL

File: datastore/tasks/tasks_test.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"golang.org/x/net/context"
+	"google.golang.org/cloud/datastore"
+
```
Comment (adg) [inline datastore/tasks/tasks_test.go]: var client ...

**Comment by adg** [comment]
LGTM modulo my naming suggestion

File: datastore/tasks/tasks_test.go (hunk 1 above)
Comment (broady) [inline datastore/tasks/tasks_test.go]: Done

**Pull request merged**